src/lib.rs

```diff
@@ -180,12 +198,19 @@ pub trait ContextMenu {
 
     /// Shows this menu as a context menu inside a [`gtk::ApplicationWindow`]
     ///
-    /// `x` and `y` are relatvie to the window top-left corner
+    /// `x` and `y` are relative to the window's top-left corner.
     #[cfg(target_os = "linux")]
     fn show_context_menu_for_gtk_window(&self, w: &gtk::ApplicationWindow, x: f64, y: f64);
     /// Get the underlying gtk menu reserved for context menus.
     #[cfg(target_os = "linux")]
     fn gtk_context_menu(&self) -> gtk::Menu;
+
+    /// Shows this menu as a context menu for the specified `NSView`.
+    ///
+    /// The menu will be shown at the coordinates of the current event
+    /// (the click which triggered the menu to be shown).
+    #[cfg(target_os = "macos")]
+    fn show_context_menu_for_nsview(&self, view: cocoa::base::id);
```

Comment:
Still want to see if it is possible to use an x & y coordinate instead of relying on the cursor position, @wusyong do you have some info about it? possibly `popUpMenuPositioningItem `

Reply:
Yes, that would be great. Unfortunately I don't currently have the time to do any further research on the coordinate issues mentioned above, but if anyone else can find a way that would be great, or perhaps it can be added later.